src/AirshipBindings.NETStandard/AirshipBindings.NETStandard/Airship.cs

```diff
@@ -102,6 +102,15 @@ public void DisplayMessageCenter()
             throw new NotImplementedException(BaitWithoutSwitchMessage);
         }
 
+        /// <summary>
+        /// Displays a specific message.
+        /// </summary>
+        /// <param name="messageId">The identifier for the message to display.</param>
+        public void DisplayMessage(string messageId)
+        {
+            throw new NotImplementedException(BaitWithoutSwitchMessage);
+        }
+
```

Comment:
Need to define the event listener in this file as well

Reply:
You should add the declaration for `OnMessageCenterUpdated` to the IAirship interface so that compiler forces us to add it here. That's another reason for us to use EventHandler instead of custom delegates – you're not allowed to define delegates in an interface because interfaces can't have type declarations. Event declarations are OK though.